topology/probes/ovsdb/ovs_of.go

```diff
@@ -329,7 +386,7 @@ func launchContinuousOnSwitch(ctx context.Context, cmd []string) (<-chan string,
 
 				return nil
 			}
-			if err := common.Retry(retry, 100, 10*time.Millisecond); err != nil {
+			if err := common.Retry(retry, 100, 500*time.Millisecond); err != nil {
```

Comment:
50 sec seems to be too long ? Is there anything special requiring to wait so long time ?

Reply:
No but 1s may be too short on a slow machine if multiple bridges are created. This is the propagation delay between the admin state of the database and the real bridge in the switch daemon. Let us choose 5s